app/presentation/flask/templates/register.html

```
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)